packages/did/src/Did.chain.ts

```diff
@@ -55,6 +55,31 @@ export async function queryDidEncoded(
   return api.query.did.did<Option<IDidChainRecordCodec>>(didIdentifier)
 }
 
+// Query ALL deleted DIDs, which can be very time consuming if the number of deleted DIDs gets large.
+export async function queryDeletedDidsEncoded(): Promise<GenericAccountId[]> {
+  const { api } = await BlockchainApiConnection.getConnectionOrConnect()
+  // Query all the storage keys, and then only take the relevant property, i.e., the encoded DID identifier.
+  return api.query.did.didBlacklist
+    .keys<GenericAccountId[]>()
+    .then((entries) =>
+      entries.map(({ args: [encodedDidIdentifier] }) => encodedDidIdentifier)
+    )
+}
+
+// Returns the raw representation of the storage entry for the given DID identifier.
+export async function queryDidDeletionStatusEncoded(
```

Comment:
Does this need to be exported?

Reply:
https://github.com/KILTprotocol/sdk-js/pull/430/commits/b55b82314ca92e4bc2c590ddc2b0bad5f3c3dfe1